exporter/loggingexporter/factory.go

```diff
@@ -44,6 +44,7 @@ func createDefaultConfig() component.Config {
 }
 
 func createTracesExporter(ctx context.Context, set exporter.Settings, config component.Config) (exporter.Traces, error) {
+	set.TelemetrySettings.Logger.Warn("The logging exporter is DEPRECATED and will be REMOVED in v0.111.0. Use the debug exporter instead: https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/debugexporter")
```

Comment:
Would it make sense to actually have a feature gate, we fail if not enabled? that way we signal even more?

Reply:
This message has been emitted for close to a year:

```
2024-09-04T09:11:09.411-0700	info	builders/builders.go:26	Deprecated component. Will be removed in future releases.	{"kind": "exporter", "data_type": "metrics", "name": "logging"}
```

But I understand this is a component that has been documented/used widely. Maybe it makes sense for this component to use a feature gate